projects/pulsar_adc_pmdz/Readme.md

```diff
@@ -36,3 +50,5 @@ Here are some pointers to help you:
   * AD7690
   * AD7982
   * AD7984
+  * AD7682
```

Comment:
We should also add the AD40XX family in the Supported parts list.

Reply:
Done.